platform/broadcom/docker-syncd-brcm.mk

```diff
@@ -17,3 +17,4 @@ $(DOCKER_SYNCD_BRCM)_RUN_OPT += -v /var/run/docker-syncd:/var/run/sswsyncd
 $(DOCKER_SYNCD_BRCM)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 
 $(DOCKER_SYNCD_BRCM)_BASE_IMAGE_FILES += bcmcmd:/usr/bin/bcmcmd
+$(DOCKER_SYNCD_BRCM)_BASE_IMAGE_FILES += bcmsh:/usr/bin/bcmsh
```

Comment:
>bcmsh [](start = 56, length = 5)

I think bcmsh script will not work in host, because the domain socket is in a different place there.

Reply:
Disagree here.  The script in the change is not the original script inside docker,  it is as below:
==
#!/bin/bash
docker exec -i syncd bcmsh "$@"
==
It calls the script inside the docker and the same domain socket is used. 
Everything works fine from host side, and people don't need go to docker to type the command.

Reply:
Thanks! I just found I wrote the wrapper:)

---
In reply to: [193163581](https://github.com/Azure/sonic-buildimage/pull/1761#discussion_r193163581) [](ancestors = 193163581)

Reply:
still marked as "Changes requested", can you remove that?